docs/pwshsupportpolicy.md

```diff
@@ -0,0 +1,30 @@
+# Pode PowerShell Version Support Policy
```

Comment:
All looks good, just one small request; could we move this file to be under the `/Getting-Started` folder, and change the title to be the following :)
```plain
# PowerShell Support Policy
```